c-pallets/segment-book/src/benchmarking.rs

```diff
@@ -83,8 +83,9 @@ benchmarks! {
     }
 
     verify_proof {
-        let v in 0 .. 20;
-
+        let v in 0 .. 20; // why 20? it is set to 50 in the pallet. Use constants
+				// This frist block is done exactly the same way in submit_challenge_prove
+				// Create a init funtion and avoid duplicating code
```

Comment:
OK, I have configured a constant for it:
https://github.com/CESSProject/cess/pull/88/files#diff-d3552cbf9043c7744c4c1073ae36fb86f92561afa8cf5a6ce99929f010619854R85